op-test

```diff
@@ -665,6 +665,22 @@ try:
         if not OpTestConfiguration.conf.args.only_flash:
             t.addTest(suites['default'].suite())
 
+    if OpTestConfiguration.conf.args.list_tests:
+        print '{0:40}'.format('Test')
+        print '{0:40}'.format('----------')
+        def print_suite(s):
+            for test in s:
+                if isinstance(test, unittest.TestSuite):
+                    print_suite(test)
+                else:
+                    id = test.id()
```

Comment:
different variable name can be used, `id` is a keyword

Reply:
good point, fixed!